### DISCUSSION TITLE
MS SQL Server versions

### DISCUSSION THREAD

**Discussion by AleksMilchov** [Budibase/budibase#7273] [Help] [answered]
Hello! 

I have a question:
The oldest version that can be used on Budibase's MS SQL Server?

**Comment by melohagan** [comment, marked as answer]
Hello @AleksMilchov 

According to this <a href="https://docs.microsoft.com/en-us/sql/connect/node-js/node-js-driver-for-sql-server?view=sql-server-ver16">docs page</a> the oldest supported version of SQL Server is 2000.
